### DISCUSSION TITLE
Does Github have client helpdesk for their issues?

### DISCUSSION THREAD

**Discussion by Dont-ask-for-it** [community/community#110911] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm talking about a simple form to submit issue and a very simple backend for client to comments on them and see updates.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Dont-ask-for-it ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
